packages/create-cloudflare/src/cli.ts

```diff
@@ -126,7 +126,7 @@ const templateMap: Record<string, TemplateConfig> = {
 		handler: runWorkersGenerator,
 	},
 	chatgptPlugin: {
-		label: `ChatGPT plugin (Typescript)`,
+		label: `ChatGPT plugin (TypeScript)`,
```

Comment:
I'm wondering if we should just remove this? The reason it was there was to delineate that it's _only_ a TS template, but doing that may create more confusion for users since we don't have a similar way to communicate that on other templates.